tests/wpt/webgl/meta/conformance/canvas/buffer-offscreen-test.html.ini

```diff
@@ -1,6 +1,4 @@
 [buffer-offscreen-test.html]
-  bug: https://github.com/servo/servo/issues/21132
```

Comment:
We're losing a number of annotations in this commit :/

Reply:
I added them back.